frontend/.gitignore

```diff
@@ -25,6 +25,3 @@ cypress/videos
 # certificates folder
 certificates/*
 !certificates/Readme.md
-
-# dotenv environment variables file
-.env
```

Comment:
.env file is needed to obtain the value of $npm_package_version.

other solutions:  

- get npm_package_version directly from package.json => security concern since the client will receive the opackage.json file
- use a node module
- Set an env-variable  that obtains $npm_package_version everytime the start or build script start (looks hacky)